Jenkinsfile

```diff
@@ -25,6 +25,9 @@ app.build(
                 memory: '10Gi',
             ]
         ],
+        storageConfig: [
+          size: '10Gi',
```

Comment:
It is already 10Gi ([default value in pac-buildkit](https://github.com/powerhome/pac-buildkit/blob/bc114c6cb82291ab400b880e250313330f2d9e03/operator/constants.py#L18-L20)), so we should increase this value.